test/integration/targets/lookup_first_found/tasks/main.yml

```
@@ -84,3 +84,7 @@
   assert:
     that:
       - "hatethisformat == '/etc/hosts'"
+
+
+# TODO: no 'terms' test
+#-- {{ query('first_found', files=['rasdaemon.yml', 'default.yml', 'q'], paths=[ '/tmp/zh', '/tmp' ], errors='ignore') }}"
```

Tests are passing locally with the two changes I suggested. This could be:

```suggestion
- name: test first_found lookup with no terms
  set_fact:
    no_terms: "{{ query('first_found', files=['missing1', 'hosts', 'missing2'], paths=['/etc'], errors='ignore') }}"

- assert:
    that: "no_terms|first == '/etc/hosts'"
```